### ISSUE TITLE
Calendar-specific settings layout broken on iPad

### ISSUE THREAD

**Issue by kristofferpeterhansel** [javalent/calendarium#102]
### Check for existing bug reports before submitting.

- [X] I searched for existing Bug Reports and found no similar reports.

### Expected Behavior

When opening the settings for a given calendar. It should be possible to navigate to the various sub-pages though the left-hand tabs.

### Current behaviour

The entire left-side tab-selector is gone.

![Screenshot 2024-03-01 at 18 23 05](https://github.com/javalent/calendarium/assets/6907530/004010ef-71bb-4dec-a50a-6bed192b5a44)

### Reproduction

This seems to be 100% consistent. Simply go to modify any calendar though one of the two  buttons (star or wrench - not really sure what whey they are both there BTW).

Settings for the calendar will open. But the left-hand pane is not there.

### Which Operating Systems are you using?

- [ ] Android
- [X] iPhone/iPad
- [ ] Linux
- [ ] macOS
- [ ] Windows

### Obsidian Version Check

1.5.8(126)

### Plugin Version

1.0.5

### Confirmation

- [ ] I have disabled all other plugins and the issue still persists.

**Comment by ebullient** [comment]
The star is for folks like me that just use a preset with no chance of accidentally changing it. It should show fewer settings.

Settings are now shown in a new pop-over modal. Clicking anywhere outside of the modal will close it.

There is a new sidebar for the popover modal that is not rendering on iOS.

**Comment by kristofferpeterhansel** [comment]
> The star is for folks like me that just use a preset with no chance of accidentally changing it. It should show fewer settings.
Okay. I twas a bit of an offhand comment just to point it out is a bit of a confusing UX. Maybe just have one button and enable more detailed editing on the general pane or something? 🤷

**Comment by ebullient** [comment]
There should be a left nav on that popover modal, and that *is* missing. So you're right.

I was just letting you know what the sparkle buttons are for (and there are some UX changes coming for that, too).

**Comment by kristofferpeterhansel** [comment]
Looks like this works for the "Custom Creator" button (although I didn't try it before. But the change looks like it would make a difference). But not when editing existing calendars.

**Comment by valentine195** [comment]
Hmm, really? It uses the same thing. How odd

**Comment by kristofferpeterhansel** [comment]
> Hmm, really? It uses the same thing. How odd

Yeah. I made sure it was 1.0.10. Reloaded it manually. Even killed force quit the app too. And still I don't see the left-hand tabs

**Comment by kristofferpeterhansel** [comment]
I am going to have to correct myself here. It works in the full editor (pencil icon?) but not the reduced editor (star icon). 

Sorry. I thought I had checked both but I must not have.

**Comment by kristofferpeterhansel** [comment]
Yes. Still happens.


https://github.com/javalent/calendarium/assets/6907530/a423690a-a73a-42d7-b642-22667824a67a
